### PR TITLE
add compatibility with juniper

### PR DESCRIPTION
**Pull request by luismorenolopera**

### CODE REVIEW

File: eox_tenant/models.py [hunk 1]
```diff
@@ -284,7 +284,8 @@ class Route(models.Model):
     )
 
     config = models.ForeignKey(
-        TenantConfig
+        TenantConfig,
+        on_delete=models.CASCADE
```
Comment (andrey-canon) [inline eox_tenant/models.py:288]: trailing comma

**Comment by felipemontoya** [comment]
@luismorenolopera have you addressed the comments by Andrey? Do you need any help to get this over the line?

**Comment by luismorenolopera** [comment]
@felipemontoya this is ready to merge

**Comment by luismorenolopera** [comment]
@felipemontoya should we add the circleci tests for python3.5 and python3.8 first before merging?

**Pull request merged**